scss/components/_button.scss

```diff
@@ -158,14 +158,17 @@ $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !defau
 }
 
 /// Adds disabled styles to a button by fading the element, reseting the cursor, and disabling pointer events.
-@mixin button-disabled($color: $primary-color) {
+@mixin button-disabled(
+  $background: $primary-color,
+  $color: $button-color
+) {
   opacity: $button-opacity-disabled;
   cursor: not-allowed;
+  background-color: $background;
+  
+  $color: color-pick-contrast($background, ($button-color, $button-color-alt));
```

Comment:
Because we do not define default semantical classes to use a11y-compliant colors. See [scss/components/_button.scss:271](https://github.com/ncoden/foundation-sites/blob/282963fef240c06f83ef093c2eb5e50bd716630d/scss/components/_button.scss#L271)

Should be:
```scss
@include button-disabled($color, auto);
```
... like at [scss/components/_button.scss:236](https://github.com/ncoden/foundation-sites/blob/282963fef240c06f83ef093c2eb5e50bd716630d/scss/components/_button.scss#L236).